command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 02:57:51PM -0700, Doug Davis wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
>
> I see no reason why a single repo can't define two specs and someone
> can be compliant with one spec but not the other.

The reason is the:

  … if “a runtime can be compliant with subsets of runtime-spec and
  still be OCI certified” is contentious at all…

section of [1].  If you feel you gain something by boxing off the
command-line API while keeping it within this repo (which I don't see,
but maybe exists), then you'd also want similar boxes around any
subset of the current runtime-spec that could be OCI certified [2].

> We get to decide this stuff.

Sure.  I'm just trying to push towards an internally-consistent
decision [3].

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72827987
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72820802
[3]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72832171